docs/Developer Resources/Custom Packages.rst

```diff
@@ -161,19 +161,37 @@ headers, or you want to build against a different kernel, you must
 specify the exact path with the *--with-linux* and *--with-linux-obj*
 options.
 
+To build RPM converted Debian packages:
+
 .. code:: sh
 
    $ cd zfs
    $ ./configure --enable-systemd
    $ make -j1 deb-utils deb-kmod
    $ for file in *.deb; do sudo gdebi -q --non-interactive $file; done
 
+To build native Debian packages:
```

Comment:
We should mention this support will first be available in the openzfs-2.2 release.

Reply:
Thanks, updated.